tasks/lint_rules/src/oxlint-rules.cjs

```diff
@@ -127,3 +127,55 @@ exports.overrideTypeScriptPluginStatusWithEslintPluginStatus = (
     rule.isNotSupported = eslintRule.isNotSupported;
   }
 };
+
+/**
+ * Some Jest rules are written to be compatible with Vitest, so we should
+ * override the status of the Vitest rules to match the Jest rules.
+ * @param {RuleEntries} ruleEntries
+ */
+exports.syncVitestPluginStatusWithJestPluginStatus = async (ruleEntries) => {
+  const vitestCompatibleRulesFile = await readFile(
+    'crates/oxc_linter/src/utils/mod.rs',
```

Comment:
Will you please add a comment over `VITEST_COMPATIBLE_JEST_RULES` saying that this file depends on it being located in this exact file?